examples/oneview_interconnect_facts.yml

```diff
@@ -66,7 +67,7 @@
     - name: Gather facts about statistics for the Interconnect that matches the specified name
       oneview_interconnect_facts:
         config: "{{ config }}"
-        name: "{{ interconnect_name }}"
+        name: interconnect_name
```

Comment:
any reason for accessing the variable like this instead of "{{ interconnect_name }}"..?

Reply:
We are getting that variable from set_fact.